RealmSwift/Object.swift

```diff
@@ -250,6 +250,16 @@ public final class DynamicObject : Object {
         }
         return nil
     }
+
+    /// :nodoc:
+    public override func valueForUndefinedKey(key: String) -> AnyObject? {
```

Comment:
Calling super would allow other properties to still work, and afaik you could call super before looking up the property in the Realm so you can get the same performance, no?